docs/docs/sdk/js.md

```diff
@@ -3,12 +3,67 @@ id: js
 title: JavaScript
 ---
 
-To install the JavaScript SDK, run:
+This generator creates TypeScript/JavaScript client that utilizes
+[axios](https://github.com/axios/axios). The generated Node module can be used
+in the following environments:
+
+Environment
+
+- Node.js
+- Webpack
+- Browserify
+
+Language level
+
+- ES5 - you must have a Promises/A+ library installed
+- ES6
+
+Module system
+
+- CommonJS
+- ES6 module system
+
+It can be used in both TypeScript and JavaScript. In TypeScript, the definition
+should be automatically resolved via `package.json`.
+([Reference](http://www.typescriptlang.org/docs/handbook/typings-for-npm-packages.html))
+
+### Building
```

Comment:
It is not clear where this should be used. Did you simply copy the README? How we build and publish the SDK is not of relevance here imo - we should just show how people can use the SDK.

Reply:
Makes sense! 
Is there documentation for that or is the old documentation still correct?
Related: https://github.com/ory/sdk/issues/22

If there is no documentation yet, what would be the best way to correct this?

Reply:
I don't think it is still correct. We use the SDK in https://github.com/ory/kratos-selfservice-ui-node